now.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 2,
+  "public": true,
+  "name": "mdx-deck",
+  "alias": "mdx-deck.jxnblk.com",
+  "builds": [
+    {
+      "src": "docs/package.json",
+      "use": "@now/static-build",
+      "config": {
+        "distDir": "dist"
+      }
+    }
+  ],
+  "routes": [
+    {
+      "src": "/main.js",
+      "dest": "/docs/main.js"
+    },
+    {
+      "src": "/(.*)",
+      "dest": "/docs/index.html"
+    }
```

Comment:
Not that I plan to immediately, but this wouldn't pick up things like static assets, would it? I wonder if this last route can only do `.html` requests?

Reply:
It seems you are correct.
I think we may have a bug on our "Output" of "/_src": 
![image](https://user-images.githubusercontent.com/7690649/54296790-48c72d80-4594-11e9-8562-e93a26ae634e.png)

Reply:
I mean, does `mdx` uses any static assets after the build?

Reply:
Yes, you are correct. But we can add routes later for that case. Something like:
```
    {
      "src": "/static/(.*)",
      "dest": "/docs/static/$1"
    }
```
But the fallback for `/docs/index.html` is still critical for a SPA.
https://github.com/zeit/now-examples/blob/master/create-react-app/now.json